cdap-watchdog/src/test/java/co/cask/cdap/logging/appender/TestResilientLogging.java

```diff
@@ -91,6 +91,8 @@ public void testResilientLogging() throws Exception {
     cConf.set(Constants.Dataset.Executor.ADDRESS, "localhost");
     cConf.setInt(Constants.Dataset.Executor.PORT, Networks.getRandomPort());
 
+    cConf.set(Constants.CFG_LOCAL_DATA_DIR, tmpFolder.newFolder().getAbsolutePath());
```

Comment:
Changes in this file are just to use a ``@ClassRule TemporaryFolder`` instead of the ``/tmp`` directory so as to not leave side-effects in ``/tmp``, right?